pkg/cmd/roachtest/roachtestutil/operations/dependency.go

```diff
@@ -29,7 +29,7 @@ func CheckDependencies(
 	for _, dep := range spec.Dependencies {
 		switch dep {
 		case registry.OperationRequiresNodes:
-			if len(c.Nodes()) == 0 {
```

Comment:
It's counter-intuitive that `c.Nodes()` denotes the empty set (of nodes). We should probably update the API; my quick audit of all the callers didn't yield any other than the above.

Reply:
Nice spot, I also ran into it, second commit on this PR fixes that issue 😉

Reply:
Yep, took me a few to realise why `OperationRequiresNodes` wasn't working.

Reply:
Thanks for catching this!